cpskin/menu/browser/menu.pt

```diff
@@ -9,7 +9,7 @@
 <ul tal:replace="structure python: view.superfish_portal_tabs()" />
 
 <a href="#" id="search-btn"></a>
-<div id="mobnav-btn">Menu de navigation</div>
+<div id="mobnav-btn" i18n:translate="">Navigation menu</div>
```

Comment:
I suppose I have to compile and add this in cpskin.locales (fr) after merging it

Reply:
@bsuttor Right now I'm doing those changes on cpskin.locales package and I'll mixing it into the French language

Reply:
@bsuttor Done with commit [9c180b7](https://github.com/IMIO/cpskin.locales/pull/3/commits/9c180b7601058ad4e446a5b984c38bdeae64d1ba)